services/movie.py

```diff
@@ -1,14 +1,19 @@
-from django.db.models import QuerySet
+from django.db import transaction
+from django.db.models import QuerySet, Q
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
+    if title:
+        queryset = Movie.objects.filter(Q(title__contains=title))
```

Comment:
Consider using `queryset.filter(Q(title__contains=title))` instead of `Movie.objects.filter(Q(title__contains=title))` to avoid resetting the queryset and improve query efficiency.